readme.md

```diff
@@ -1077,6 +1077,17 @@ Some commonly asked for features are available as modules:
 
 *Warning: The `webpack` function is executed twice, once for the server and once for the client. This allows you to distinguish between client and server configuration using the `isServer` property*
 
+Multiple configurations can be combined together with function composition. For example:
+
+```js
+module.exports = withTypescript(withSass({
```

Comment:
it would probably be interesting to show an example with a *compose* function like [`_.flowRight`](https://lodash.com/docs/4.17.5#flowRight) for example

Reply:
Feel free to add it as extra example 🙏

Reply:
My personal preference would be http://ramdajs.com/docs/#pipe but I'm not convinced this would do anything to add clarity to the example. If someone did not know `_.flowRight` or `R.pipe` they might be more confused than a straight up JS example.

Having said that I think the overall approach is a little clunky and would like to propose something a bit more functional. I'd prefer to be able to do this:

```
module.exports = R.pipe(
 withTypeScript(applyCustomWithTypeScriptOverrides),
 withSass(applyCustomWithTypeScriptOverrides),
 () => ({
   webpack(config, options) {
      // Further custom configuration here
      return config
    }
 })
)
```